sample_apps/simple_app_under_keystone/instructions.md

```diff
@@ -8,12 +8,26 @@ No dependencies on anything, can build on any architecture and with any compiler
 
 ## Example App
 
+Start Qemu
+
+1. `docker pull keystoneenclaveorg/keystone:dev`
+2. `docker run -it --entrypoint /bin/bash keystoneenclaveorg/keystone:dev --volume ABS_CERTIFIER_PATH:/certifier`
+3. `cd /keystone/build && ./scripts/run-qemu.sh`
+   1. Note down the `PORT` Qemu uses
+4. Username: `root` Password: `sifive`
+5. `insmod keystone-driver.ko`
+
 Bare app -- keystone_api only, without certifier:
 
-1. Set up Keystone: todo instructions. Working on updating Docker image.
-2. `cd certifier/sample_apps/simple_app_under_keystone && make -f keystone_example_app.mak bare_app`
+1. `docker ps # find the hash of your Keystone container`
+2. `docker -it CONTAINER_HASH /bin/bash`
+3. `cd /certifier/sample_apps/simple_app_under_keystone`
```

Comment:
I think you would need `source /keystone/source.sh`

Reply:
resolved